CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/ArrangementPluginRegistry.java

```diff
@@ -50,6 +50,8 @@
  */
 public class ArrangementPluginRegistry {
 
+    public static final String LOCKPOSITIONS = LockVertexPositionsPlugin.class.getName();
```

Comment:
A minor one but could you call this `LOCK_POSITIONS` for better readability?

Reply:
Ive actrually refactored to use the terms PINNED and UNPINNEd to better represent icons used.